src/reset.css

```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```

Comment:
I'd use a global selector here rather than list so many elements. Then any selectors for specific elements below the global selector will override the global settings.